### PR TITLE
feat: order-form add checkout

### PR DESCRIPTION
**Pull request by defaultProps**

### CODE REVIEW

File: src/locales/en.json [hunk 1]
```diff
@@ -19,6 +19,7 @@
       "pair": "Pair",
       "pay": "Pay",
       "max": "MAX",
+      "insuffcientBalance": "insuffcient sUDT balance",
```
Comment (DylanOpenDev) [inline src/locales/en.json]:
```suggestion
      "insuffcientBalance": "Insuffcient sUDT balance",
```

File: src/locales/zh.json [hunk 2]
```diff
@@ -10,6 +10,7 @@
       "testWallet": "钱包测试",
       "balances": "余额",
       "connecting": "正在连接钱包...",
+      "insuffcientBalance": "sUDT余额不足",
       "inUse": "已使用",
       "free": "余额",
       "HexAddressTooltip": "当前以太坊地址。其他人可以发送CKB到这个地址使用钱包支持密码锁。",
```
Comment (DylanOpenDev) [inline src/locales/zh.json:22]: Why are some of the first letters of these variables uppercase, and some of the first letters are lowercase?

File: src/pages/Trade/components/TradePairOrder/index.tsx [hunk 3]
```diff
@@ -17,14 +18,55 @@ export default () => {
...
   const [buyer, seller] = Order.pair
+  const [disabled, setDisabled] = useState(false)
+  const [submitText, setSubmitText] = useState(i18n.t(`trade.placeOrder`))
 
   const changePair = () => {
     setVisiblePopver(false)
     Order.togglePair()
     form.resetFields()
   }
+  // dai -> ckb 卖单
```
Comment (DylanOpenDev) [inline src/pages/Trade/components/TradePairOrder/index.tsx]: Please write annotations with English?

File: src/pages/Trade/components/TradePairOrder/index.tsx [hunk 4]
```diff
@@ -17,14 +18,55 @@ export default () => {
...
+  const checkPay = (_: any, value: string): Promise<void> => {
```
Comment (DylanOpenDev) [inline src/pages/Trade/components/TradePairOrder/index.tsx:37]: If the first variable has useless, please remove it.

File: src/pages/Trade/components/TradePairOrder/styled.tsx [hunk 5]
```diff
@@ -58,18 +58,25 @@ export const PairOrderFormBox = styled.div`
...
     .submitBtn {
+      padding: 0;
       font-size: 17px;
       font-weight: 500;
       height: 60px;
       width: 100%;
       color: rgba(0, 106, 151, 1);
       text-align: center;
     }
+    .submitBtn[disabled] {
```
Comment (DylanOpenDev) [inline src/pages/Trade/components/TradePairOrder/styled.tsx]: Why some of the class names are in camel case and some are in underscore form?

File: src/locales/en.json (hunk 1 above)
Comment (defaultProps) [reply src/locales/en.json]: it's fixed.

File: src/locales/zh.json (hunk 2 above)
Comment (defaultProps) [reply src/locales/zh.json:22]: it's fixed.

File: src/pages/Trade/components/TradePairOrder/index.tsx (hunk 3 above)
Comment (defaultProps) [reply src/pages/Trade/components/TradePairOrder/index.tsx]: it's deleted

File: src/pages/Trade/components/TradePairOrder/index.tsx (hunk 4 above)
Comment (defaultProps) [reply src/pages/Trade/components/TradePairOrder/index.tsx:37]: it's fixed

File: src/pages/Trade/components/TradePairOrder/styled.tsx (hunk 5 above)
Comment (defaultProps) [reply src/pages/Trade/components/TradePairOrder/styled.tsx]: Checke all files className and use the camel case.

File: src/locales/zh.json (hunk 2 above)
Comment (DylanOpenDev) [reply src/locales/zh.json:22]: `HexAddressTooltip`'s first letter is still uppercase and the others are lowercase

File: src/pages/Trade/components/TradePairConfirm/index.tsx [hunk 6]
```diff
@@ -21,9 +21,12 @@ export default function TradePairConfirm() {
   const Wallet = useContainer(WalletContainer)
   const Order = useContainer(OrderContainer)
   const [buyer, seller] = Order.pair
+  const [disabled, setDisabled] = useState(false)
 
   const onConfirm = async () => {
-    // eslint-disable-next-line no-debugger
+    setDisabled(true)
+    // eslint-disable-next-line no-console
+    console.log(Order.pay)
```
Comment (DylanOpenDev) [inline src/pages/Trade/components/TradePairConfirm/index.tsx:29]: Please remove `console.log`

File: src/pages/Trade/components/TradePairOrder/index.tsx (hunk 4 above)
Comment (DylanOpenDev) [reply src/pages/Trade/components/TradePairOrder/index.tsx:37]: I mean `_: any`

File: src/pages/Trade/components/TradePairOrder/index.tsx [hunk 7]
```diff
@@ -1,38 +1,95 @@
...
+      setFieldPay(false)
+      return Promise.reject(i18n.t(`trade.insuffcientCKBBalance`))
+    }
+
+    setFieldPay(true)
+
+    return Promise.resolve()
+  }
+
+  const checkPrice = (_: any, value: string): Promise<void> => {
```
Comment (DylanOpenDev) [inline src/pages/Trade/components/TradePairOrder/index.tsx:65]: What does the `_: any` mean?

File: src/pages/Trade/components/TradePairConfirm/index.tsx [hunk 8]
```diff
@@ -21,9 +21,12 @@ export default function TradePairConfirm() {
   const Wallet = useContainer(WalletContainer)
   const Order = useContainer(OrderContainer)
   const [buyer, seller] = Order.pair
+  const [disabled, setDisabled] = useState(false)
 
   const onConfirm = async () => {
-    // eslint-disable-next-line no-debugger
+    setDisabled(true)
+    // eslint-disable-next-line no-console
```
Comment (defaultProps) [inline src/pages/Trade/components/TradePairConfirm/index.tsx]: 。

File: src/pages/Trade/components/TradePairOrder/index.tsx [hunk 9]
```diff
@@ -1,38 +1,95 @@
...
-  const [disabled] = useState(false)
   const [buyer, seller] = Order.pair
 
+  // disabled button
+  const [fieldPay, setFieldPay] = useState(false)
+  const [fieldPrice, setFieldPrice] = useState(false)
+
+  const MIN_PRICE = 1 / 10 ** 10
+  const MIN_VAL = 1 / 10 ** (Order.orderType === OrderType.Buy ? 8 : 10)
+  const MIN_ORDER = Order.orderType === OrderType.Buy ? 147 : 289
```
Comment (defaultProps) [inline src/pages/Trade/components/TradePairOrder/index.tsx]: uses util/const file

File: src/pages/Trade/components/TradePairOrder/index.tsx (hunk 4 above)
Comment (defaultProps) [reply src/pages/Trade/components/TradePairOrder/index.tsx:37]: no, it's necessary. this is antd form paramater required.

File: src/pages/Trade/components/TradePairOrder/index.tsx (hunk 7 above)
Comment (defaultProps) [reply src/pages/Trade/components/TradePairOrder/index.tsx:65]: this is antd form component  necessary paramter required.

File: src/pages/Trade/components/TradePairOrder/index.tsx (hunk 4 above)
Comment (defaultProps) [reply src/pages/Trade/components/TradePairOrder/index.tsx:37]: no , it's mecessary paramter  require.

Comment (DylanOpenDev) [reply src/pages/Trade/components/TradePairOrder/index.tsx:37]: OK. I think it's very strange

**Review by DylanOpenDev — APPROVED** [review]

**Pull request merged**